app/src/components/buttons/network-list-button.tsx

```diff
@@ -32,6 +33,7 @@ export const NetworkListButton = (props: Props) => {
           {network.chainName}
         </Text>
       </Row>
+      <Text textSize='14px'>${Number(gasEstimate).toFixed(4)}</Text>
```

Comment:
Might result in `NaN` maybe we should require the prop type to be `number`. That, or use the network prop to let this component fetch the gasEstimates

Reply:
The component now uses the `useNetworkFee` hook directly.